ncm-nrpe/src/main/perl/nrpe.pod

```diff
@@ -11,80 +11,84 @@ way.
 
 =head1 COMPONENT STRUCTURE
 
-All fields are mandatory, unless it is explicitly stated:
+All fields are required (but most have sensible defaults) unless otherwise stated:
 
 =over 4
 
-=item * /software/components/nrpe/log_facility : string
+=item * C<< /software/components/nrpe/options/log_facility : string >>
 
 The syslog facility that should be used for logging purposes.
 
-=item * /software/components/nrpe/pid_file : string
+=item * C<< /software/components/nrpe/options/pid_file : string >>
 
 File in which the NRPE daemon should write it's process ID number.
 
-=item * /software/components/nrpe/server_port : type_port
+=item * C<< /software/components/nrpe/options/server_port : type_port >>
 
 The port the daemon will listen to.
 
-=item * /software/components/nrpe/server_address ? string
+=item * C<< /software/components/nrpe/options/server_address ? string >>
 
 Address that nrpe should bind to if you do not want nrpe to bind on all interfaces.
 
 Optional field.
 
-=item * /software/components/nrpe/nrpe_user : string
+=item * C<< /software/components/nrpe/options/nrpe_user : string >>
 
 User the daemon will run as. For instance, 'nagios'.
 
-=item * /software/components/nrpe/nrpe_group : string
+=item * C<< /software/components/nrpe/options/nrpe_group : string >>
 
 Group the daemon will run as. For instance, 'nagios'.
 
-=item * /software/components/nrpe/allowed_hosts : type_hostname[]
+=item * C<< /software/components/nrpe/options/allowed_hosts : type_hostname[] >>
 
 List of hosts allowed to order the NRPE daemon to run commands.
 
-=item * /software/components/nrpe/dont_blame_nrpe : boolean
+Must be specified, no default is provided.
```

Comment:
i would put the options that have no default and are required on top of the pod (because those must be configured)